repository/Seaside-Tools-Web.package/monticello.meta/categories.st

```diff
@@ -1 +1 @@
-self packageOrganizer ensurePackage: #'Seaside-Tools-Web' withTags: #()!
+SystemOrganization addCategory: #'Seaside-Tools-Web'!
```

Comment:
This reverts a change from an earlier format of Pharo. I think we should revert this particular change in this PR.